maintenance-page/production.tfvars

```
@@ -0,0 +1,2 @@
+aws_region             = "eu-west-2"
+cloudfront_domain_name = "backoffice.production.406beacons.com"
```

```suggestion
cloudfront_domain_name = "production.406beacons.com"
```
not sure if this is on purpose to not mess with production while testing this?

---

Yep - thats a mistake.. copied over the back office setup and this didn't get changed.